src/app/loadout-builder/generated-sets/CompareDrawer.tsx

```diff
@@ -224,7 +224,7 @@ export default function CompareDrawer({
             {setItems.map((item) => (
               <div key={item.bucket.hash} className={styles.item}>
                 <ConnectedInventoryItem item={item} />
-                <Sockets item={item} lockedMods={loSetAssignedMods.get(item.id)} size="small" />
+                <Sockets item={item} lockedMods={loSetAssignedMods[item.id]} size="small" />
```

Comment:
I generally prefer objects to maps with string keys

Reply:
Yeah I use them more then. It's just the possibility of undefined values that get hidden by typescript have caught me out a few times, maps always have the possibility of that undefined.